.ci/install_crio.sh

```diff
@@ -64,14 +64,9 @@ then
 fi
 
 echo "Get CRI Tools"
-critools_repo="${kubernetes_sigs_org}/cri-tools"
-go get "$critools_repo" || true
-pushd "${GOPATH}/src/${critools_repo}"
-critools_version=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
-git checkout "${critools_version}"
-go install ./cmd/crictl
-sudo install "${GOPATH}/bin/crictl" /usr/bin
-popd
+crictl_version=v1.14.0
```

Comment:
also, can you please move this to https://github.com/kata-containers/tests/blob/master/versions.yaml and then get the value here using something like: `crictl_version=$(get_test_version "external.crictl.version")`

Reply:
Yes, I will add that.